packages/wrangler/src/__tests__/cert.test.ts

```diff
@@ -202,11 +202,15 @@ describe("wrangler", () => {
 						expires_on: oneYearLater.toISOString(),
 					});
 
-					const cert = await uploadMTlsCertificate("some-account-id", {
-						certificateChain: "BEGIN CERTIFICATE...",
-						privateKey: "BEGIN PRIVATE KEY...",
-						name: "my_cert",
-					});
+					const cert = await uploadMTlsCertificate(
+						undefined,
```

Comment:
Most tests below have just been updated to add an undefined `complianceConfig` parameter since that signals we are not in FedRAMP High mode.

Reply:
I have switched all tests to use `COMPLIANCE_REGION_UNKNOWN`.